src/lib/datepicker/datepicker.ts

```diff
@@ -134,7 +134,12 @@ export class MatDatepickerContent<D> extends _MatDatepickerContentMixinBase
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.None,
 })
+
 export class MatDatepicker<D> implements OnDestroy, CanColor {
+
```

Comment:
nit: no blank line

Reply:
done in a88fecbec2c3292017613bf4e75c38ff7c632042